input/input.c

```diff
@@ -894,6 +911,99 @@ void mp_input_set_mouse_pos(struct input_ctx *ictx, int x, int y)
     input_unlock(ictx);
 }
 
+static int find_touch_point_index(struct input_ctx *ictx, int id)
+{
+    for (int i = 0; i < ictx->num_touch_points; i++) {
+        if (ictx->touch_points[i].id == id)
+            return i;
+    }
+    return -1;
+}
+
+static void notify_touch_update(struct input_ctx *ictx)
+{
+    // queue dummy cmd so that touch-pos can notify observers
+    mp_cmd_t *cmd = mp_input_parse_cmd(ictx, bstr0("ignore"), "<internal>");
+    queue_cmd(ictx, cmd);
```

Comment:
this seems to be necessary because `mouse-pos` is married to `MP_EVENT_INPUT_PROCESSED` which is triggered in `mp_process_input`. could be improved some day..